js_modules/dagit/packages/core/src/assets/AssetTable.tsx

```diff
@@ -138,105 +156,142 @@ const AssetEntryRow: React.FC<{
-              >
-                <Button>View in Asset Graph</Button>
-              </Link>
-            ) : (
-              <Button disabled={true}>View in Asset Graph</Button>
-            )}
-            <Popover
-              position="bottom-right"
-              content={
-                <Menu>
-                  <MenuLink
-                    text="View details…"
-                    to={`/instance/assets/${path.join('/')}`}
-                    icon="view_list"
-                  />
-                  <MenuItem
-                    text="Wipe Asset…"
-                    icon="delete"
-                    disabled={!canWipe}
-                    intent="danger"
-                    onClick={() => canWipe && onWipe(assets)}
+    const onChange = (e: React.FormEvent<HTMLInputElement>) => {
+      if (e.target instanceof HTMLInputElement) {
+        const {checked} = e.target;
+        const shiftKey =
+          e.nativeEvent instanceof MouseEvent && e.nativeEvent.getModifierState('Shift');
+        onToggleChecked({checked, shiftKey});
+      }
+    };
+
+    const liveData = asset && liveDataByNode[toGraphId(asset.key)];
+    const stepKey = asset && asset.definition?.opNames[0];
+
+    return (
+      <tr>
+        <td style={{paddingRight: '4px'}}>
+          <Checkbox checked={isSelected} onChange={onChange} />
+        </td>
+        <td>
+          <AssetLink
+            path={path}
+            url={linkUrl}
+            isGroup={isGroup}
+            icon={isGroup ? 'folder' : asset?.definition ? 'asset' : 'materialization'}
+          />
+          <Description>
+            {asset?.definition &&
+              asset.definition.description &&
+              markdownToPlaintext(asset.definition.description).split('\n')[0]}
+          </Description>
+        </td>
+        <td>
+          {asset?.definition && (
+            <Box flex={{direction: 'column', gap: 2}}>
+              <RepositoryLink
+                showIcon
+                showRefresh={false}
+                repoAddress={{
+                  name: asset.definition.repository.name,
+                  location: asset.definition.repository.location.name,
+                }}
+              />
+            </Box>
+          )}
+        </td>
+        <td>
+          {liveData && stepKey ? (
+            liveData.lastMaterialization ? (
+              <Mono>
+                <AssetRunLink
+                  runId={liveData.lastMaterialization.runId}
+                  event={{stepKey, timestamp: liveData.lastMaterialization.timestamp}}
+                >
+                  <TimestampDisplay
+                    timestamp={Number(liveData.lastMaterialization.timestamp) / 1000}
+                    timeFormat={{showSeconds: false, showTimezone: false}}
                   />
-                </Menu>
-              }
+                </AssetRunLink>
+              </Mono>
+            ) : (
+              <span>–</span>
+            )
+          ) : undefined}
```

Comment:
Why sometimes a `–` and sometimes undefined?

Reply:
Hmm I made the column blank until the data loads in because it comes in separately from the asset name. We could put a bunch of small spinners in there maybe...